documentation/source/simulator_support.rst

```diff
@@ -94,6 +94,19 @@ To enable HDL code coverage, add Verilator's coverage option(s) to the :make:var
 
 This will result in coverage data being written to ``coverage.dat``.
 
+Waveforms
+---------
+
+To get waveforms in VCD format, add Verilator's trace option(s) to the
+:make:var:`EXTRA_ARGS` make variable, for example:
+
+  .. code-block:: make
+
+    EXTRA_ARGS += --trace
```

Comment:
```suggestion
    EXTRA_ARGS += --trace --trace-structs
```

Reply:
Why ""--trace-structs", to show that we can add several option in EXTRA_ARGS ?

Reply:
Yeah, and to make users aware of the typical arguments they care about for tracing.

Note also that on line 109 below you already list both arguments, so it would be consistent to do so here too.

Reply:
Also, that's what ``VERILATOR_TRACE`` did (which we are replacing with this documentation section).